lib/redis-client.js

```diff
@@ -40,9 +40,12 @@ RedisClient.prototype.getLink = function(link) {
   })
 }
 
-RedisClient.prototype.getLinks = function() {
+RedisClient.prototype.getLinks = function(searchString) {
```

Comment:
Hmm, since Code Climate is still complaining about the complexity here, perhaps we need to extract a new `SearchHelper` class or some such. It can be private to this module. Mind doing that? Or I can do it in a separate PR.

Reply:
I will do it here, want a clean run 😄